### DISCUSSION TITLE
How can I link Day Planner to a static (not daily) file?

### DISCUSSION THREAD

**Discussion by feedhopper** [ebullient/obsidian-day-planner-og#19] [Q&A]
Hi there,

it's GREAT to see this plugin being picked up close to it's original concept.

What I would love is another planner mode:

**Static File** would just link to a static file without generating any dailies or relying on this to be done by external plugin.

Why do I want this?

I worked with dailies but found that it cluttered my workspace quite a lot. Also, I had difficulties finding stuff on the occasions that I wanted to go back to some note without rememberingthea specific date.

What I use is something different now:

1) I have various notes where I put tasks depending on area.
2) I created a document sitting in sidebar that displays part of these notes depending on due or started date as well as tags.
3) for displaying them, I use callouts that form a kind of time blocking system.

I always wished for a plugin to do exactly that. Day planner is the only one coming close.

Is there any chance to make this plugin link to a static file?
I would be VERY grateful indeed!

**Comment by ebullient** [comment]
Try [command mode](https://github.com/ebullient/obsidian-day-planner-og#day-planner-mode). You can use that to bind to any note that has a Day Planner heading in it.

I happen to have daily notes, but I use command mode to define when the Day Planner plugin is bound to a day (basically only within working hours). I have outside scripts in a cron job that run bind/unbind commands..
